apps/lib/apps.c

```diff
@@ -539,6 +492,8 @@ X509_REQ *load_csr(const char *file, int format, const char *desc)
     X509_REQ *req = NULL;
     BIO *in;
 
+    if (desc == NULL)
+        desc = "CRL";
```

Comment:
This doesn't look right.

Reply:
Yeah, always this confusion of CSRs and CRLs :wink: - fixed.